### ISSUE TITLE
xClusterPreferredOwner: The resource is moving cluster group in Set-TargetResource

### ISSUE THREAD

**Issue by johlju** [PowerShell/xFailOverCluster#64]
When setting preferred owner on a cluster group or a cluster resource it moves the cluster group to the first node. In a production environment this could be a bad thing.

See [line 98-99](https://github.com/PowerShell/xFailOverCluster/blob/dev/DSCResources/MSFT_xClusterPreferredOwner/MSFT_xClusterPreferredOwner.psm1#L98-L99)

```
        Write-Verbose -Message "Moving Cluster Group $ClusterGroup to node $($Nodes[0])"
        $null = (Get-ClusterGroup -cluster $ClusterName) | Where-Object {$_.name -like $ClusterGroup} | Move-ClusterGroup -Node $Nodes[0]
```

Why is it moving the cluster group? And if this is a must, then I think this should be handled differently.

**Comment by johlju** [comment]
Is it meant to move to the preferred owner, if that owner is not the preferred? But that should not be up to the resource, would it?

**Comment by bgelens** [comment]
I don't think you want a resource to fight with what the cluster service is doing. Just have it configure the setting and leave it alone would be best IMO.

**Comment by johlju** [comment]
I agree.
